src/widgets/TextWidget.tsx

```diff
@@ -71,6 +72,7 @@ export interface TextWidgetStyle extends CommonStyleProps {
   textShadowColor?: ColorProp;
   textShadowRadius?: number;
   textShadowOffset?: { height: number; width: number };
+  decorationLine?: 'none' | 'line-through' | 'underline';
```

Comment:
We try to follow react native props.
Please rename this to `textDecorationLine`, and the values should be `'none' | 'underline' | 'line-through' | 'underline line-through'` . See https://reactnative.dev/docs/text-style-props#textdecorationline